components/brave_wallet_ui/components/desktop/popup-modals/balance_details_modal/balance_details_modal.style.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from 'styled-components'
+import * as leo from '@brave/leo/tokens/css'
+import Icon from '@brave/leo/react/icon'
+
+// Shared Styles
+import { Column, Text } from '../../../shared/style'
+
+export const BalancesColumn = styled(Column)`
+  background-color: ${leo.color.page.background};
+  border-radius: 16px;
+  padding: 24px;
+  gap: 16px;
+`
+
+export const SectionText = styled(Text)`
+  line-height: 22px;
+  color: ${leo.color.text.secondary};
+  margin-right: 4px;
```

Comment:
can we use `padding` on the rows that contain this instead?

Reply:
Applied a `gap` to the `row` instead!